releases/nightly-build/compose-files/docker-compose-nexus-arm64.yml

```diff
@@ -250,13 +246,9 @@ services:
       EDGEX_SECURITY_SECRET_STORE: "true"
       KONGURL_SERVER: kong
       REGISTRY_HOST: edgex-core-consul
-      SECRETSERVICE_CACERTPATH: /tmp/edgex/secrets/ca/ca.pem
       SECRETSERVICE_SERVER: edgex-vault
-      SECRETSERVICE_SNIS: edgex-kong
```

Comment:
Technically speaking, this is a Kong TLS related override, not a Vault TLS override.